securedrop/sdconfig.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+import config as _config
+
+MYPY = False
+if MYPY:
+    from typing import List  # noqa: F401
+
+
+class SDConfig(object):
+    def __init__(self):
+        # type: () -> None
+        self.FlaskConfig = _config.FlaskConfig
+        self.JournalistInterfaceFlaskConfig = \
+            _config.JournalistInterfaceFlaskConfig
+        self.SourceInterfaceFlaskConfig = _config.SourceInterfaceFlaskConfig
+
+        self.ADJECTIVES = ""
+        self.DATABASE_ENGINE = ""
+        self.DEFAULT_LOCALE = ""
```

Comment:
You'll want to set this to `en_US` as a default since the current logic in the application is:

```
getattr(config, 'DEFAULT_LOCALE', 'en_US')
```

to handle the case where `DEFAULT_LOCALE` was not set in existing instances

Reply:
Okay.